sscanss/app/widgets/graphics.py

```diff
@@ -1101,6 +1101,12 @@ def __init__(self, point, *args, size=6, **kwargs):
         self.default_pen = QtGui.QPen(QtGui.QColor(), 0)
         self.highlight_pen = QtGui.QPen(QtGui.QColor(), 0, QtCore.Qt.PenStyle.DashLine)
         self.setFlag(QtWidgets.QGraphicsItem.GraphicsItemFlag.ItemSendsGeometryChanges)
+        self.row_highlighted = False
```

Comment:
would be better to call this "highlighted" or "selected" since the GraphicsPointItem does not have any rows